experimental/PropertyDDS/packages/property-changeset/src/test/array.spec.ts

```diff
@@ -956,14 +956,8 @@ describe("Array Operations", function () {
 									? "no inserts"
 									: `inserts ${insertNames.join(", ")}`;
 							it(title, () => {
-								const insertsA: (
-									| [number, ReturnType<typeof generateNamedEntities>]
-									| never
-								)[] = [];
-								const insertsB: (
-									| [number, ReturnType<typeof generateNamedEntities>]
-									| never
-								)[] = [];
+								const insertsA: [number, ReturnType<typeof generateNamedEntities>][] = [];
+								const insertsB: [number, ReturnType<typeof generateNamedEntities>][] = [];
```

Comment:
@jason-ha - The inferred type of the original empty array '[]' is 'never[]', but you don't need to preserve the 'never' when specifying the array type here.

Reply:
Agreed. When I saw your change I went to look to see who had added the `never` :p -- I just don't know.